### DISCUSSION TITLE
Need BMGS (Board, Medium, Grade, Subject) master data for all tenant to resource type

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Knowlg/Community#167] [Q&A]
Need BMGS (Board, Medium, Grade, Subject) master data for all tenant to resource type. 
can you help us how to get this data either API or Database. Can you pls share API which can be used to fetch this data or any other approach like to share any query so we can fetch the data directly from database.

**Comment by pallakartheekreddy** [comment]
@kamal-singhal Can you please provide more details? We didn't get what do you mean by master data for all tenant to resource type

**Reply by pallakartheekreddy** [reply]
@kamal-singhal any update on this?

**Reply by kamal-singhal** [reply]
![BMGS-Resource](https://github.com/Sunbird-Knowlg/Community/assets/125978084/823e4bf1-e713-4e35-89cc-b43d5aaf1680)
Hi,
Actually when we create any new resource then BMGS value coming in drop down. we need to make a excel sheet with all values of BMGS to all tenant for resource content. so pls suggest how to fetch that data in one go. 
for your ref. i attached the screen shot.

**Reply by maheshkumargangula** [reply]
@kamal-singhal — What is the use case to use BMGS of all the tenants while creating the resource. Can you please explain the requirement in detail?

Below are the options we have to fetch all tenants' BGMS values.
- Please consider using the Search API and apply a filter on "Term" to get all the terms in the sunbird instance. With this, you will lose the dependency information among the terms of a framework.

**Reply by kamal-singhal** [reply]
Thanks for the response @maheshkumargangula.

Actually client require the all BMGS data tenant wise for resource type content. that 'why we need this data. 
As you suggested to fetch the data through search API but we require BMGS data tenant wise.

**Reply by pallakartheekreddy** [reply]
@kamal-singhal As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
